docs/connecting_to_remote_db_aws_aurora.md

```diff
@@ -0,0 +1,101 @@
+# Connecting to a remote Klaxon database in an AWS Aurora cluster
+
+This documentation is primarily for first-time setup to connect a locally running app to a remote Aurora cluster database. This doc assumes several things:
```

Comment:
i think we'd benefit from a brief line here about why one would want to do this in the course of local development